inst/admiral-lookup-book.csv

```diff
@@ -48,26 +85,67 @@ ADEG,BDS,ABLFL,Derive analysis baseline flag,admiral,restrict_derivation,https:/
 ADEG,BDS,ANL01FL,Flag last result within an analysis visit and analysis timepoint for post-baseline records,admiral,restrict_derivation,https://pharmaverse.github.io/admiral/reference/restrict_derivation.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
 ADEG,BDS,CHG,Derive change from baseline,admiral,derive_var_chg,https://pharmaverse.github.io/admiral/reference/derive_var_chg.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
 ADEG,BDS,PCHG,Derive percent change from baseline,admiral,derive_var_pchg,https://pharmaverse.github.io/admiral/reference/derive_var_pchg.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEX,OCCDS,TRTEDTM,Derive treatment end datetime,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
-ADEX,OCCDS,TRTSDTM,Derive treatment start datetime,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
-ADEX,OCCDS,TRTSDT,Derive treatment start date ,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
-ADEX,OCCDS,ASTDTM,Derive analysis start datetime,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
-ADEX,OCCDS,AENDTM,Derive analysis end datetime,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
+ADEX,OCCDS,TRTSDT,Derive treatment start date,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
+ADEX,OCCDS,AENDTM,Derive analysis end datetime and date/time imput. flags,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
```

Comment:
```suggestion
ADEX,OCCDS,AENDTM,Derive analysis end datetime and date/time impute. flags,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adex.R
```

Reply:
Apologies, missed these ones. So during our conversation it was if, let's say, AENDTM is derived in ADEX and you get imputations (AENDTF and AEDNTMF), we capture this as three variables:
1) AENDTM  - Derive analysis end datetime
2) AENDTF - Derive analysis end date imputation flag
3) AENTMF - Derive analysis end time imputation flag

I have added imputation flags as new variables, but have not adjusted the datetime variable descriptions correctly.

Reply:
![Screenshot 2024-07-15 at 11 10 21 AM](https://github.com/user-attachments/assets/20ca2571-faaa-479c-9a34-0147ec9c79dc)

@jsaltane123 thanks Julija so much for the work. I'm going through the spreadsheet now. Just wanted to follow up with this comment about AENDTM variables across ADAE, ADCM, ADEX, ADPC, and ADPPK